pyproject.toml

```diff
@@ -21,7 +21,7 @@ classifiers = [
 # list package dependencies here
 dependencies = [
   "copylot @ git+https://github.com/czbiohub-sf/coPylot",
-  "iohub==0.1.0",
+  "iohub @ git+https://github.com/czbiohub-sf/iohub.git",
```

Comment:
I think Talon is writing something but pin this to `0.2.0a1`

Reply:
Done